backend/btrixcloud/operator.py

```diff
@@ -1002,6 +1008,13 @@ async def sync_crawl_state(
                 # get next file done
                 file_done = await redis.lpop(self.done_key)
 
+            page_crawled = await redis.lpop(f"{crawl.id}:{self.pages_key}")
+
+            while page_crawled:
+                page_dict = json.loads(page_crawled)
+                await self.page_ops.add_page_to_db(page_dict, crawl.id)
```

Comment:
Can pass crawl.oid from the CrawlSpec here

Reply:
Done